Cooper/ClientApp/src/app/components/games/games.component.css

```diff
@@ -1,74 +1,65 @@
-/* esComponent's private CSS styles */
-.games {
-    margin: 0 0 2em 0;
-    list-style-type: none;
-    padding: 0;
-    width: 15em;
+@import url('https://fonts.googleapis.com/css?family=Roboto:900');
```

Comment:
it is main font for all application and should be setted in common file in common selector, in other case it will be dublicate in different parts of application

Reply:
Was settled